### ISSUE TITLE
Node ports not accessible - Can't create channels to node - ISP at fault

### ISSUE THREAD

**Issue by dawiepoolman** [raspibolt/raspibolt#198]
Hi guys

TL;DR:  My ISP had my ports 8333, 9735, 18333, 19735 blocked.
They had to assign me a new public WAN IP.  Dont go mad for days assuming the issue is with the setup if it could be as simple as your ISP blacking ports.

I have DMed Stadicus on the bird site (Twitter) and we decided to create an issue for others to follow if they experience a similar issue..

I manged to get the RPi3 node up and running using the awesome walkthrough and I even managed to create outbound channels from my RPi3 node on testnet.  I have toggled to mainnet for testing but my node but it does not seem to be visible on the network e.g. via sites like http://canyouseeme.org/ or https://bitnodes.earn.com/ or https://1ml.com/.  
It once briefly appeared under my nic 'FederalReverse' on 1ML and also when I searched for nodes and channels using eclair's desktop app connected on testnet.

I have dedicated an internal IP for the RPi using router DHCP and the blockchain is downloading just fine and staying in sync.

I have forwarded the ports on the fiber router and made sure there are no firewall blocks.
On the RPi the ufw firewall settings have also been set to allow the ports as per walkthrough for the mainnet and testnet (now inactive) ports.

I have attached screenshots as well of my config.

I have logged a call with my ISP to confirm that the ports are not blocked on the ISP site.  I have also scanned with Zenmap on my laptop but the ports seemed to be closed.  I can ping my external public IP though.

As I wrote this issue a NOC engineer from my ISP confirmed the issue was on their side.  I got a new IP address now.

💃 

<img width="946" alt="routerfirewall" src="https://user-images.githubusercontent.com/2351212/45252406-f5122800-b355-11e8-89d1-15a7a03bfd43.PNG">
<img width="957" alt="routerportforwarding" src="https://user-images.githubusercontent.com/2351212/45252407-f5122800-b355-11e8-9b82-a28839c47240.PNG">
<img width="607" alt="rpi3firewall" src="https://user-images.githubusercontent.com/2351212/45252408-f5aabe80-b355-11e8-8486-386c984ed8d8.PNG">
